azure/app/parameters/production.template.json

```diff
@@ -141,6 +141,28 @@
     },
     "ROLLBAR_ENV": {
       "value": "production"
+    },
+    "GOVUK_VERIFY_ENABLED": {
+      "value": ""
+    },
+    "VSP.VERIFY_ENVIRONMENT": {
+      "value": "PRODUCTION"
+    },
+    "VSP.SAML_SIGNING_KEY": {
+      "reference": {
+        "keyVault": {
+          "id": "${keyVaultId}"
+        },
+        "secretName": "VspSamlSigningKey"
+      }
+    },
+    "VSP.SAML_PRIMARY_ENCRYPTION_KEY": {
+      "reference": {
+        "keyVault": {
+          "id": "${keyVaultId}"
+        },
+        "secretName": "VspSamlPrimaryEncryptionKey"
+      }
```

Comment:
If these secrets are missing from the Key Vault, the deployment will fail. I suspect, if we want to be able to push this to `production` after merge, we need to set these to an empty string for now.

Reply:
Same applies to `test`.

Reply:
I've added them to the `production` KeyVault. We don't have a `test` KeyVault yet

Reply:
Ah, I meant changing the parameters file to be `"value": ""` for now. I think that's better than adding placeholder values to a production Key Vault.

Reply:
Ahh ok, yeah that's better. I'll change it